src/lib/routes.ts

```diff
@@ -33,9 +33,15 @@ export const routes = {
                 values: () => apiJoin(routes.api.base, 'mentorship/update/value'),
                 profile: () => apiJoin(routes.api.base, 'mentorship/update/profile'),
             },
+            connect: {
+                // We can add other paths here like booking sessions once cal.com integration is done
```

Comment:
I added connect because I can see us adding more URLs around booking a set date/time, sending, reminders, etc. 

Not 100% sure if we should have this pattern now without the extra calls or wait to make this update later.

Reply:
The first step is just getting them all in the same place! :+1: